traitsui/examples/demo/Advanced/Tabular_editor_demo.py

```diff
@@ -55,39 +55,39 @@
 - If the demo is running under QT, you can move rows by clicking and dragging.
 
 Hopefully, this simple example conveys some of the power and flexibility that
-the :py:class:`TabularAdapter` class provides you. But, just in case it
```

Comment:
The `:py:class:` (and other `:py:*:` tags) don't work in the standalone etsdemo application because the code has no access to documentation.